internal/cmd/kafka/command_link_list.go

```diff
@@ -104,3 +111,18 @@ func getListFields(includeTopics bool) []string {
 
 	return append(x, "SourceClusterId", "DestinationClusterId", "State", "Error", "ErrorMessage")
 }
+
+func getMirrorTopicNames(kafkaREST *pcmd.KafkaREST, clusterId, linkName string) ([]string, error) {
+	listMirrorTopicsResponseDataList, httpResp, err := kafkaREST.CloudClient.ListKafkaMirrorTopicsUnderLink(clusterId, linkName)
+	if err != nil {
+		return nil, kafkarest.NewError(kafkaREST.CloudClient.GetUrl(), err, httpResp)
```

Comment:
does this function catch the detail in http response?

Reply:
If the error is a `GenericOpenAPIError`, then it decodes and returns the `message` field.